Project_source/app/src/main/java/android/clicker/school_live_simulator/NullComputerCourseState.kt

```diff
@@ -0,0 +1,20 @@
+package android.clicker.school_live_simulator
+
+open class NullComputerCourseState: ComputerCourseState {
+    override price: Int = 0
```

Comment:
Same thing in every class
```suggestion
    override val price: Int = 0
```

Reply:
Fixed